android/sound.cpp

```diff
@@ -325,15 +349,17 @@ oboe::DataCallbackResult CSound::onAudioOutput ( oboe::AudioStream* oboeStream,
 
     QMutexLocker locker ( &MutexAudioProcessCallback );
 
-    std::size_t to_write = numFrames*oboeStream->getChannelCount();
+    std::size_t to_write = numFrames * oboeStream->getChannelCount();
```

Comment:
Do we have an issue / PR to get all these fixed, too?

Reply:
> Do we have an issue / PR to get all these fixed, too?

Are you referring to the coding style (it looks like this on Github) or are you referring to a Github security warning (which I just assume you are seing)? @softins recently had a PR for improving the latter: #1385.

I don't know of a generic issue to track it though. Can you open one?

Reply:
The assignment warning, yes.  I thought I'd seen one...

Reply:
I've meanwhile opened #1428 and I'm marking this comment as resolved.